composer.json

```diff
@@ -25,8 +25,8 @@
     "phpstan": "build-cs/vendor/bin/phpstan analyse --memory-limit=2048M -c phpstan.neon.dist",
     "format": "build-cs/vendor/bin/phpcbf --standard=phpcs.xml.dist --report-summary --report-source",
     "lint": "build-cs/vendor/bin/phpcs --standard=phpcs.xml.dist",
-    "test": "phpunit -c phpunit.xml.dist --verbose",
-    "test-multisite": "phpunit -c tests/multisite.xml --verbose"
+    "test": "phpunit -c phpunit.xml.dist --verbose --testsuite performance-lab",
+    "test-multisite": "phpunit -c tests/multisite.xml --verbose --testsuite performance-lab"
```

Comment:
I guess this would be solved just by adding another `--testsuite` arg to the end so that it overrides whatever is set here initially? So if I wanted to run `auto-sizes` tests locally, I could just do:

```sh
npm run test-php -- -- --  --testsuite auto-sizes
```

This would result in somewhat unexpected output:

```
> phpunit -c phpunit.xml.dist --verbose --testsuite performance-lab '--testsuite' 'auto-sizes'
```

But it would successfully only run the `auto-sizes` test suite (as long as the `break` is removed as I suggested below).

Reply:
Do we need to make any updates based on this feedback?

Reply:
I think this is resolved, yes.